src/components/Collections/Table/Table.vue

```diff
@@ -311,36 +343,15 @@ export default {
     savePlioDetails(entryIndex, plioDetails) {
       // save the plio details after they are fetched from the PlioListItem
 
-      // to enable the search functionality on things like plio title, status, date etc,
-      // those details need to be stored in the filteredData object and that too, inside the "name"
-      // key as that key contains the details of plios
+      // Each plio's details are being stored in the filteredData object and that too,
+      // inside the "name" key as that key contains the details of plios
       if (this.filteredData != undefined && this.filteredData[entryIndex] != undefined) {
         this.filteredData[entryIndex]["name"] = {
           ...this.filteredData[entryIndex]["name"],
           ...plioDetails,
         };
```

Comment:
is this even needed anymore? because search is happening on the server?

Reply:
Which i think is overkill, not super effective

Reply:
TODO